eventlogger/src/main/java/gov/cms/ab2d/eventlogger/ApiRequestEventLogger.java

```diff
@@ -0,0 +1,29 @@
+package gov.cms.ab2d.eventlogger;
+
+import lombok.Data;
+
+import java.time.OffsetDateTime;
+
+/**
+ * Class to create and log an API request coming from a user
+ */
+@Data
+public class ApiRequestEventLogger implements LoggableEvent {
+    // id
+    private Long id;
```

Comment:
It looks like each and every implementation of `LoggableEvent` has the `id` field, which means it could be a good candidate for moving to an abstract base class?